bio_hansel/main.py

```diff
@@ -200,6 +201,11 @@ def main():
     df_md = None
     if args.scheme_metadata:
         df_md = read_metadata_table(args.scheme_metadata)
+    if scheme == 'typhi':
```

Comment:
I like this approach to integrating existing scheme metadata into the subtyping metadata dataframe. Would it be possible to extend this to all built-in typing schemes? I'm not sure if `resource_filename` would play nice with missing files, but you might be able to wrap it in a `try/except` block:

```python
try:
    ...
    df_md = pd.concat([df_md, read_metadata_table(resource_filename(program_name, f'data/{scheme}/{scheme}_scheme_metadata.tsv'))], axis=1)
    ...
except Exception: # probably a better idea to catch a ResourceNotFoundException or whatever it is
    pass
```

Reply:
Ya I'll put that on my list of things to do and get to it. I think that is a much more elegant solution then including the case just for typhi as I currently have it

Reply:
Done with commit 6a5e0e4, it now should work with any added files should they be called "metadata.tsv". 

I had to create the file path with some basic string concatenation as I couldn't get it to work with the {}.

Reply:
You need the `f` before the string to perform string literal interpolation `f'data/{scheme}/metadata.tsv'` where `scheme` is a variable that exists in that scope. (see https://www.python.org/dev/peps/pep-0498/ for more info). It's a Python 3.6+ feature as an alternative to `'data/{scheme}/metadata.tsv'.format(scheme=scheme)` or using `+` to build up your strings.

Reply:
Oh I see, I missed that. I'll give that a go as it looks way better then what I have